content/en/news/releases/1.9.x/announcing-1.9/_index.md

```diff
@@ -0,0 +1,185 @@
+---
+title: Announcing Istio 1.9.0
+linktitle: 1.9
+subtitle: Major Update
+description: Istio 1.9 release announcement.
+publishdate: 2021-02-09
+release: 1.9.0
+skip_list: true
+aliases:
+    - /news/announcing-1.9
+    - /news/announcing-1.9.0
+---
+
+We are pleased to announce the release of Istio 1.9.
+
+{{< relnote >}}
+
+## Changes
```

Comment:
Remove this file and I'll with my PR https://github.com/istio/istio.io/pull/8910 I'll fill in its contents. Take all review comments comments and put them in `content/en/news/releases/1.9.x/announcing-1.9/change-notes/index.md`

Reply:
Ok

Reply:
Done.